playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -1248,6 +1246,7 @@ public void setProfiler(Profiler profiler) {
     private void configureSubtitleView() {
         SubtitleView exoPlayerSubtitleView = null;
         if(exoPlayerView != null) {
+            exoPlayerView.setSubtitleViewPosition(playerSettings.getSubtitleStyleSettings().getSubtitlePosition());
```

Comment:
@GouravSna 
should check if thee expression here is not null then execute the logic